.github/workflows/deploy-docs.yaml

```
@@ -4,12 +4,15 @@ on:
   push:
     branches:
       - main
+      - beta/*
     paths:
       - mkdocs.yaml
       - "**/*.md"
       - "**/*.svg"
       - "**/*.png"
       - "**/*.jpg"
+    tags:
+      - \d+.\d+
```

Could you add `v*` as well? :pray: 
If you prefer, changing it to like `v[0-9]+*` is okay for me.

```suggestion
      - v*
      - \d+.\d+
```